### ISSUE TITLE
[BUG] Replace `cupy.core` usages with `cupy`

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/clx#389]
**Describe the bug**

There are a few usages of `cupy.core` in the codebase here. `cupy.core` is viewed as internal by CuPy and is marked as such in CuPy 9. The recommendation is to use `cupy` instead as things in `cupy.core` that are intended to be public are made available there anyways (for example `cupy.core.ndarray` would be changed to `cupy.ndarray`). Making these kinds of changes remains compatible with older CuPy versions as well.

**Comment by jakirkham** [comment]
Looks like this is limited to docstrings

```python
python/clx/analytics/periodicity_detection.py:    :rtype: cupy.core.core.ndarray
python/clx/analytics/periodicity_detection.py:    :rtype: cupy.core.core.ndarray
python/clx/analytics/periodicity_detection.py:    :type prdg: cupy.core.core.ndarray
python/clx/analytics/periodicity_detection.py:    :rtype: cupy.core.core.ndarray
```

**Comment by jakirkham** [comment]
PR ( https://github.com/rapidsai/clx/pull/410 ) should fix this

**Comment by efajardo-nv** [comment]
Closed by #410.
